package-lock.json

```diff
@@ -66,11 +66,11 @@
       }
     },
     "node_modules/@babel/generator": {
-      "version": "7.18.12",
```

Comment:
is there a reason this file is being changed? Can we revert changes here?

Reply:
I have no idea. I didn't change. if needed how can we fix it?